pkg/media/output.go

```diff
@@ -60,14 +62,25 @@ type AudioOutput struct {
 	codec livekit.AudioCodec
 }
 
-func NewVideoOutput(codec livekit.VideoCodec, layer *livekit.VideoLayer) (*VideoOutput, error) {
+func NewVideoOutput(codec livekit.VideoCodec, layer *livekit.VideoLayer, maxEncodedWith, maxEncodedHeight uint32) (*VideoOutput, error) {
 	e, err := newVideoOutput(codec)
 	if err != nil {
 		return nil, err
 	}
 
 	e.logger = logger.GetLogger().WithValues("kind", "video", "layer", layer.Quality.String())
 
+	threadCount, outQueueLen := getVideoEncoderThreadCountAndOutputBufferSize(layer, maxEncodedWith, maxEncodedHeight)
+	e.logger.Infow("video layer", "width", layer.Width, "height", layer.Height, "threads", threadCount, "queueLength", outQueueLen)
+
+	queueIn, err := gst.NewElement("queue")
```

Comment:
might eventually help with debugging to give all your queues names

Reply:
Indeed. I just added names.